apps/arweave/src/ar_inflation.erl

```diff
@@ -50,40 +50,30 @@ calculate_post_15_y1_extra() ->
 pre_15_calculate(Height) when Height =< ?REWARD_DELAY ->
 	1;
 pre_15_calculate(Height) ->
-	?WINSTON_PER_AR
-		* 0.2
-		* ?GENESIS_TOKENS
-		* math:pow(2, -(Height - ?REWARD_DELAY) / ?PRE_15_BLOCK_PER_YEAR)
-		* math:log(2)
-        / ?PRE_15_BLOCK_PER_YEAR.
+	?WINSTON_PER_AR * 0.2 * ?GENESIS_TOKENS *
+	math:pow(2, -(Height - ?REWARD_DELAY) / ?PRE_15_BLOCK_PER_YEAR) *
+	math:log(2) / ?PRE_15_BLOCK_PER_YEAR.
 
 calculate_base(Height) ->
 	{Ln2Dividend, Ln2Divisor} = ?LN2,
 	Dividend = Height * Ln2Dividend,
 	Divisor = ?BLOCKS_PER_YEAR * Ln2Divisor,
 	Precision = ?INFLATION_NATURAL_EXPONENT_DECIMAL_FRACTION_PRECISION,
 	{EXDividend, EXDivisor} = ar_fraction:natural_exponent({Dividend, Divisor}, Precision),
-	?GENESIS_TOKENS
-		* ?WINSTON_PER_AR
-		* EXDivisor
-		* 2
-		* Ln2Dividend
-		div (
-			10
-			* ?BLOCKS_PER_YEAR
-			* Ln2Divisor
-			* EXDividend
-		).
+	?GENESIS_TOKENS * ?WINSTON_PER_AR * EXDivisor * 2 * Ln2Dividend div (
+		10 *
+		?BLOCKS_PER_YEAR *
+		Ln2Divisor *
+		EXDividend
+	).
 
 calculate_base_pre_fork_2_5(Height) ->
-	?WINSTON_PER_AR
-		* (
-			0.2
-			* ?GENESIS_TOKENS
-			* math:pow(2, -(Height) / ?BLOCK_PER_YEAR)
-			* math:log(2)
-		)
-		/ ?BLOCK_PER_YEAR.
+	?WINSTON_PER_AR * (
```

Comment:
I'm fine if we have a convention of `*` at the end or the beginning of a line, but I think we should leave line breaks in equations and long lists/tuples upt to the engineer

Reply:
it's now newlined for each math operation